apps/nextjs-website/react-components/types/StripeLink/StripeLink.types.ts

```diff
@@ -0,0 +1,9 @@
+import { EIconProps } from "../../components/common/EIcon";
+import { CtaButtonProps, Generic } from "../common/Common.types";
+
+export interface StripeLinkProps {
+  theme: 'dark' | 'light';
+  subtitle: Generic | string;
+  icon?: EIconProps;
+  readonly buttonText?: ReadonlyArray<CtaButtonProps | Generic>;
```

Comment:
```suggestion
  buttonText?: ReadonlyArray<CtaButtonProps | Generic>;
```
`readonly` isn't needed